tests/git_edit_index_tests.py

```diff
@@ -274,6 +289,18 @@ def test_calls_correct_git_command_and_returns_correct_status(self):
             universal_newlines=True
         )
 
+    def test_calls_correct_git_command_with_ignored_files(self):
+        STATUS = 'status'
+        self.subprocess.check_output.return_value = STATUS
+
+        status = git_status(True)
```

Comment:
I suggest changing
```
status = git_status(True)
```
to
```
status = git_status(show_ignored=True)
```
to make the meaning of the `True` argument more apparent. Also, this will make the test more robust as we way add other parameters to `git_status()` in the future.